virttest/qemu_devices/qcontainer.py

```diff
@@ -1763,9 +1763,9 @@ def pcic_by_params(self, name, params):
                 name, bus_type, 'xio3130-downstream', name)
             driver = 'x3130-upstream'
         else:
-            if driver == 'pci-bridge':  # addr 1-19, chasis_nr
+            if driver == 'pci-bridge':  # addr 1-31, chasis_nr
```

Comment:
This is a bit confusing comment as qemu would fail on `31` as it assumes hex by default. Let's change the comment to `0x01 - 0x1f` (or `0-1f`)